easybuild/easyblocks/b/binutils.py

```diff
@@ -166,8 +166,22 @@ def install_step(self):
                           os.path.join(self.installdir, 'include', 'libiberty.h'))
 
             if not glob.glob(os.path.join(self.installdir, 'lib*', 'libiberty.a')):
-                copy_file(os.path.join(self.cfg['start_dir'], 'libiberty', 'libiberty.a'),
-                          os.path.join(self.installdir, 'lib', 'libiberty.a'))
+                # Be a bit careful about where we install into
+                libdir = os.path.join(self.installdir, 'lib')
+                if os.path.exists(libdir) and os.path.isdir(libdir):
+                    if not os.listdir(libdir):
+                        # At this point the lib directory should be populated, if not try the other option
+                        libdir = os.path.join(self.installdir, 'lib64')
+                else:
+                    # If the directory doesn't exist, try the other option
+                    libdir = os.path.join(self.installdir, 'lib64')
```

Comment:
This can be simplified a bit:

```python
# At this point the lib directory should exist and be populated, if not try the other option
if not (os.path.exists(libdir) and os.path.isdir(libdir) and os.listdir(libdir)):
    libdir = os.path.join(self.installdir, 'lib64')

    
```